src/_includes/install/php-versions-template.md

```diff
@@ -2,10 +2,6 @@ Supported PHP versions:
 
 {% for version in supported_php_versions %}
```

Comment:
Shouldn't we add a new composer.lock file for 2.4 in this PR? I know it won't be the stable release version, but at least it will generate the correct version of PHP for 2.4 at beta.